test/specs/core.tooltip.tests.js

```diff
@@ -732,20 +732,8 @@ describe('Core.Tooltip', function() {
 			// Trigger an event over top of the element
 			var pointIndex = 1;
 			var datasetIndex = 0;
-			var meta = chart.getDatasetMeta(datasetIndex);
-			var point = meta.data[pointIndex];
-			var node = chart.canvas;
-			var rect = node.getBoundingClientRect();
-			var evt = new MouseEvent('mousemove', {
```

Comment:
There are a lot of other places in this file where we're creating a `new MouseEvent` instead of calling `triggerMouseEvent`. Do they need to be updated as well?